test/test_main.py

```diff
@@ -182,8 +182,10 @@ def _read_and_clear_state(self):
         return state
 
     def _simulate_event(self, event_spec):
+        ppath = Path(__file__).parent
         env = {
-            'PATH': "{}:{}".format(Path(__file__).parent / 'bin', os.environ['PATH']),
+            'PATH': "{}:{}".format(ppath / 'bin', os.environ['PATH']),
+            'PYTHONPATH': str(ppath.parent),
```

Comment:
Is this the *only* directory that should be in pythonpath? It seems to assume a system install of PyYAML, and not a local venv, etc.
I would actually think we want to do the same that we did for PATH, and include ppath.parent as the first entry, but include the os.environ['PYTHONPATH'] along with it.

Reply:
I can do that :-)
in my testing it wasn't needed, with yaml in a venv or out of it (because the venv tweaks the builtin paths), but i see no harm in doing it